microprofile/messaging/src/test/java/io/helidon/microprofile/messaging/AdHocConfigBuilderTest.java

```diff
@@ -76,6 +132,38 @@ void customValueOverride() {
         assertThat(c.getValue(TEST_KEY, String.class), is(TEST_TOPIC_CUSTOM));
     }
 
+    @Test
+    void putAllTestMp() {
+        Map<String, String> propMap = Map.of(
+                "mp.messaging.outcoming.test-channel." + TEST_KEY, TEST_TOPIC_CONFIG
+        );
+
+        Map<String, String> propMap2 = Map.of(
+                "mp.messaging.connector." + TEST_CONNECTOR + "." + ADDITION_ATTR_1, ADDITION_ATTR_1_VALUE,
+                "mp.messaging.connector." + TEST_CONNECTOR + "." + ADDITION_ATTR_2, ADDITION_ATTR_2_VALUE
+        );
+
+        resolver.registerConfig(resolver.getBuilder()
+                                        .withSources(MpConfigSources.create(propMap))
+                                        .build(), null);
+
+        Config config = (Config) resolver.getConfig();
+
+        resolver.registerConfig(resolver.getBuilder()
+                                        .withSources(MpConfigSources.create(propMap2))
+                                        .build(), null);
```

Comment:
`io.helidon.config.mp.MpConfigProviderResolver#registerConfig` has no defense against `null` and registered config is stored in `io.helidon.config.mp.MpConfigProviderResolver#CONFIGS` map with `null` key, but that Config is not acessible any more because `getConfig` has defense against `null`
```java
    public Config getConfig(ClassLoader loader) {
        if (null == loader) {
            loader = ClassLoader.getSystemClassLoader();
        }

```

Reply:
Fixed all instances to work the same - if loader is `null`, use context class loader.